### DISCUSSION TITLE
How to get a list of icons provided in pulsar

### DISCUSSION THREAD

**Discussion by hareldvd** [pulsar-edit/.github#190] [General]
This seems to me as a dumb question that I failed to figure out myself. Sorry for that.
Using flex-tool-bar I edit the toolber list (~/.pulsar/toolbar.cson) and use icons for it's buttons, however I can't remember the names I should use for those.
Is there a way to display those icons?

**Comment by savetheclocktower** [comment]
The **Styleguide: Show** command will open a window with a bunch of UI elements. One of the sections is dedicated to icons and includes the names of all the icons.

<img width="940" alt="Screenshot 2023-06-13 at 11 28 46 AM" src="https://github.com/pulsar-edit/.github/assets/3450/1b9c735f-7c45-4383-84b7-3820fb7a9937">

**Comment by hareldvd** [comment]
Working great. Just added this command on the flex toolbar I mentioned.
Thanks.
